### ISSUE TITLE
PHP bindings are obsolete and should be sunsetted

### ISSUE THREAD

**Issue by gsmecher** [ketiltrout/getdata#12]
The PHP bindings are fairly out-of-date and should be removed (likely) or fixed (less likely).

The TSRMLS macros (TSRMLS_FETCH, TSRMLS_CC, TSRMLS_DC) are discussed in an accepted PHP RFC here:

https://wiki.php.net/rfc/native-tls

This was unanimously endorsed for a proposed PHP7 release (2015). The I think these macros were then carried forward until PHP8 as inert, and were removed at PHP8 (2021):

https://www.seidengroup.com/2020/12/07/porting-extensions-to-php-8/

I would be happy to submit a PR that removes PHP support entirely. I did check to see if a simple search-and-replace could quickly fix the build failures but ended up with a test that segfaulted, indicating deeper trouble than I was willing to debug.

**Comment by ketiltrout** [comment]
Yeah, there are a number of bindings that probably should be retired.  Let me see if I can figure out why we wrote the PHP bindings in the first place...
